site/docs/README.md

```diff
@@ -0,0 +1,40 @@
+# AIID Documentation Index
+
+1. [Application Development Guide](../gatsby-site/README.md)
+
+Local development setup, testing procedures, etc.
+
+3. [Database Architecture](DATABASE.md)
+
+MongoDB collections structure, schema details, and data relationships
+
+4. [Database Migration Guide](MIGRATIONS.md)
+
+Database migration procedures and configuration
+
+5. [Deployment Guide](DEPLOYMENT.md)
+
+Deployment procedures and configuration for Netlify, Prismic, and other services
+
+6. [Internationalization Guide](i18N.md)
+
+Internationalization system documentation and translation process
+
+7. [Taxonomy Guide](TAXA.md)
+
+Guide for implementing and managing taxonomies in the system
+
+8. [Frontend Documentation](FRONTEND.md)
+
+Details about the frontend development process, including the use of Tailwind CSS and Flowbite.
+
+9. [API Documentation](API.md)
+
+Details about the AIID API, including how to access and interact with it, and information on email notifications.
+
+## Getting Help
+
+1. Open issues on [GitHub](https://github.com/responsible-ai-collaborative/aiid/issues)
+2. Visit the [contact page](https://incidentdatabase.ai/contact)
+3. Join the project's Slack (link available on contact page)
```

Comment:
I don't see the Slack link on the contact page https://incidentdatabase.ai/contact